include/ord/Timing.h

```diff
@@ -104,6 +107,7 @@ class Timing
   bool isEndpoint_(sta::Pin* sta_pin);
   float getPinSlew_(sta::Pin* sta_pin, MinMax minmax);
   float getPinArrival_(sta::Pin* sta_pin, RiseFall rf, MinMax minmax);
+  float getPinSlack_(sta::Pin* sta_pin, RiseFall rf, MinMax minmax);
```

Comment:
Why is there an underscore in the name?

Reply:
This was just a convention that I had followed, indicating that the function is a private version of the user-facing public API. These private functions can access internal objects like STA pins. If there is a better coding style, I can change it; please let me know what would be appropriate.

Reply:
I would just drop the underscore as the overload resolution is pretty clear.  If you prefer a different name then I would replace underscore with Priv or Impl.